pkg/deploy/deploy.go

```diff
@@ -87,65 +87,19 @@ func deploy(client dtclient.Client, apis api.APIs, em *entityMap, c *config.Conf
 
 	case config.SettingsType:
 		log.Info("\tDeploying config %s", c.Coordinate)
-		return deploySetting(client, properties, renderedConfig, c)
+		return deploySetting(clientSet.settings(), properties, renderedConfig, c)
 
 	case config.ClassicApiType:
-		log.Info("\tDeploying config %s", c.Coordinate)
-		return deployConfig(client, apis, em, properties, renderedConfig, c)
+		return deployConfig(clientSet.classic(), apis, em, properties, renderedConfig, c)
```

Comment:
nit: should we rename `deployConfig` to `deployClassicConfig` ? 
It could clear up the naming overlap with `config` used for "monaco configuration object" (e.g. in the logs), and be in line with the "classic" naming of the ApiType, client and deploy/classic.go